projects/neural_correlations/EXP2-HighOrder/NeuCorr_Exp2.ipynb

```diff
@@ -133,30 +164,58 @@
     "                plt.ylabel(\"Frequency\")\n",
     "                plt.savefig(\"cellsHist_\" + str(ts))\n",
     "                plt.close()\n",
-    "                print \"++ Computing entropy ++\"\n",
     "                entropyX.append(ts)\n",
-    "                entropyY.append(computeEntropy(subSpikeTrains))                \n",
-    "                \n",
-    "                print \"++ Analyzing correlations (whole columns) ++\"                \n",
-    "                subSpikeTrains = subSampleWholeColumn(spikeTrains, colIndices, tm.getCellsPerColumn(), ts)\n",
+    "                entropyY.append(computeEntropy(subSpikeTrains))   \n",
+    "\n",
+    "                #print \"++ Analyzing correlations (whole columns) ++\"\n",
+    "                ### First the LARGE subsample of columns:\n",
+    "                subSpikeTrains = subSampleWholeColumn(spikeTrains, colIndicesLarge, tm.getCellsPerColumn(), ts)\n",
     "                (corrMatrix, numNegPCC) = computePWCorrelations(subSpikeTrains, removeAutoCorr=True)\n",
```

Comment:
see above comments for experiment 1, I think this still contains a mixture of cell pairs (within same column or across columns)